iron-selectable.html

```diff
@@ -306,11 +284,6 @@
           this._updateSelected();
         }
       }.bind(this));
```

Comment:
I think you can factor away the bind here if you want. The callback is called in the context of the observed node and you shouldn't need the argument now.